main.js

```diff
@@ -37,198 +37,187 @@ function checkWindowSize() {
-card5.innerHTML = `
-  <div class="mycard">
-    <h3 class="profesional-Art-heading">
-      Profesional Art Printing Data
-    </h3>
-    <p class="profesional-Art-content">
-      A daily
-      selection of privately personalized reads;
-      no accounts or sign-ups required. has been the
-      industry's standard
-    </p>
-    <ul class="profesional-Art-ul">
-      <li class="profesional-Art-li">html</li>
-      <li class="profesional-Art-li">bootstrap</li>
-      <li class="profesional-Art-li">Ruby</li>
-    </ul>
-  </div>
-  <a href="#6" id="hide-me" class="profesional-Art-btm hide button popup">See Project</a>
-`;
-
-const card6 = document.createElement('div');
-card6.className = 'art-container2 project-card';
-card6.innerHTML = `
-  <div class="mycard">
-    <h3 class="profesional-Art-heading">
-      Profesional Art Printing Data
-    </h3>
-    <p class="profesional-Art-content">
-      A daily
-      selection of privately personalized reads;
-      no accounts or sign-ups required. has been the
-      industry's standard
-    </p>
-    <ul class="profesional-Art-ul">
-      <li class="profesional-Art-li">html</li>
-      <li class="profesional-Art-li">bootstrap</li>
-      <li class="profesional-Art-li">Ruby</li>
-    </ul>
-  </div>
-  <a href="#7" id="hide-me" class="profesional-Art-btm button popup">See Project</a>
-`;
-
-// Append the div elements to the section with class "work-container"
+// Array of project data
+const projects = [
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
```

Comment:
- [ ] You have done well so far, :+1: :100: But I think there are still some requirements you need to meet before moving to the next milestone.
- - As the previous reviewer suggested, You are meant to have one array of projects where you loop through and display projects on the page and also show more details with the pop-up when the button is clicked, currently, you tend to have two arrays, One for projects that display on the homepage and one for the pop-up, That should not be the case, I'll suggest you delete the `projects` array and then loop through `data` array to display them on the homepage, and also bring the **popup to show more details of that given project when the `see project` button is clicked** :smile: :wave: